src/SFA.DAS.EmployerIncentives.Web/Controllers/ApplicationCompleteController.cs

```diff
@@ -1,36 +1,37 @@
 ﻿using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Options;
-using SFA.DAS.EmployerIncentives.Web.Infrastructure.Configuration;
 using SFA.DAS.EmployerIncentives.Web.Services.Applications;
 using SFA.DAS.EmployerIncentives.Web.Services.LegalEntities;
 using SFA.DAS.EmployerIncentives.Web.ViewModels.ApplicationComplete;
 using System;
 using System.Threading.Tasks;
+using SFA.DAS.EmployerIncentives.Web.Services.BankDetails;
 
 namespace SFA.DAS.EmployerIncentives.Web.Controllers
 {
     [Route("{accountId}/application-complete/{applicationId}")]
     public class ApplicationCompleteController : ControllerBase
     {
         private readonly IApplicationService _applicationService;
-        private readonly ExternalLinksConfiguration _configuration;
+        private readonly IBankDetailsStatusService _bankDetailsStatusService;
+        private const string VrfStatusCompleted = "Case Request completed";
 
         public ApplicationCompleteController(ILegalEntitiesService legalEntitiesService,
                                              IApplicationService applicationService,
-                                             IOptions<ExternalLinksConfiguration> configuration)
+                                             IBankDetailsStatusService bankDetailsStatusService)
             : base(legalEntitiesService)
         {
+            _bankDetailsStatusService = bankDetailsStatusService;
             _applicationService = applicationService;
-            _configuration = configuration.Value;
         }
 
         [HttpGet]
         [Route("")]
         public async Task<IActionResult> Confirmation(string accountId, Guid applicationId)
         {
             var application = await _applicationService.Get(accountId, applicationId, includeApprenticeships: false);
-            var legalEntityName = await GetLegalEntityName(accountId, application.AccountLegalEntityId);
-            var model = new ConfirmationViewModel(accountId, application.AccountLegalEntityId, legalEntityName, _configuration.ManageApprenticeshipSiteUrl);
+            var legalEntity = await _bankDetailsStatusService.RecordBankDetailsComplete(accountId, application.AccountLegalEntityId);
```

Comment:
this doesn't feel quite right

Reply:
RecordBankDetailsComplete sounds like a command, so shouldn't return anything. can LE be fetched separately after completing the command?

Reply:
That's been changed in the latest commit